.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "workbench.colorTheme": "Default High Contrast Light",
+  "workbench.colorTheme": "Abyss",
```

Comment:
You have severely worsened the VSCode experience!!

Reply:
why did that get put in there?, also, WHY DO YOU HAVE A LIGHT THEME FOR YOUR CODE???

Reply:
It's the best theme of VSCode! Together with Wide Latin, it's the ultimate coding environment.

Reply:
it also severely worsens MY VSCode experience! because I go blind on light mode

Reply:
This is incorrect. Default High Contrast Light is the way of life.

Reply:
it hurts me :(

Reply:
No.

Reply:
it's french bread

Reply:
No, it's not a baguette, it's the theme of life.

Reply:
![image](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/88983354/d4652932-5dc2-4f66-8a4e-540eb48bbab3)

Reply:
"Dark" themes that are "good for your eyes" and "good for battery life" or some stuff are terrible. Default High Contrast Light is the best theme on earth.

Reply:
Alright, another argument: As we all know, light attracts bugs.

Take a look at this dark mode code.

![dark mode code](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/130385691/32881deb-d073-433c-b4aa-7d6afb3f5e6c)

As you can see, the background is dark, and the code is light. So, the code attracts bugs, and the code is buggy.

Now, look at this code with the superior theme:

![light mode code](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/130385691/328683ea-5bde-4954-98ba-bb4df4abe202)

As you can see, the background is light, and the code is dark. So, the background attracts bugs, and they don't care for the code, so the code's not buggy.

Reply:
The white background can't handle many bugs, also it takes all the bugs away from the one with the dark background

Reply:
Well, let's solve that problem by just getting rid of dark themes entirely?

Reply:
All the bugs go to the WHITE background, not the DARK BLUE AND PURPLE, text

Reply:
Sure, but it still doesn't go to the code. The white background distracts the bugs.

Reply:
the magnets holding the code to the wall is not that strong, if too many bugs fly to it it will fall in the trash can

Reply:
Luckily, there are no magnets involved.

Reply:
acually this is on a fridge